src/compas_robots/configuration.py

```diff
@@ -343,25 +343,24 @@ def from_prismatic_and_revolute_values(cls, prismatic_values, revolute_values, j
         )
 
     @property
-    def data(self):
-        """dict : The data representing the configuration.
+    def dtype(self):
+        return "compas_robots/Configuration"
```

Comment:
ah is this because we normally assume 2nd level import?

Reply:
yep, exactly